macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -368,8 +414,13 @@ open class AvatarView: NSView {
 	/// the maximum number of initials to be displayed when we don't have an image
 	static let maximumNumberOfInitials: Int = 2
 
-	/// the color used for the outline view
-	static let outlineColor = NSColor(named: "AvatarView/outlineColor", bundle: FluentUIResources.resourceBundle)!
+	/// the color used for the border
+	static let defaultBorderColor = NSColor(named: "AvatarView/borderColor", bundle: FluentUIResources.resourceBundle)!
+
+	static let borderWidth: CGFloat = 2.0
```

Comment:
Is the ring thickness always the same value irrespective of the Avatar size? (same for the contentInset below)

Reply:
currently that's what i did because a) we don't have size ramp. b) i don't have exact design for the ring :(